en/config/safety.md

```diff
@@ -227,6 +229,44 @@ Parameter | Description
 [VT_FW_ALT_ERR](../advanced_config/parameter_reference.md#VT_FW_ALT_ERR) | Maximum negative altitude error for fixed wing flight. If the altitude drops more than this value below the altitude setpoint the vehicle will transition back to MC mode and enter failsafe RTL.
 
 
+## Safety Switches {#safety_switch}
+
+A safety switch allows you to immediately crash or return the vehicle from the remote control transmitter
+(if you lose control of the vehicle, this may be better than allowing it to continue flying).
+
+This section lists the available safety switches.
+
+### Kill Switch {#kill_switch}
+
+A kill switch immediately stops all motor outputs, causing the vehicle to crash.
+
+The kill switch is enabled as part of *QGroundControl* [Flight Mode Setup](../config/flight_mode.md#safety_switch).
+
+
+### Arm/Disarm Switch {#arming_switch}
+
+> **Tip** The arming switch is similar to the [kill switch](#kill_switch), but cannot be enabled in all modes during flight (i.e. those modes where it less likely to be needed).
+
+An arm/disarm switch immediately disarms (stop) motors for those [flight modes](../getting_started/flight_modes.md) that *support disarming in flight* (i.e. for these modes it is the same as the [kill switch](#kill_switch)). This includes the modes: 
```

Comment:
I would phrase this whole section differently: An arming switch is not really an alternative to a kill switch. It is a replacement for the default stick (dis)arming (and thus also used for arming). And it serves the exact same purpose: making sure there is an intentional step involved before the motors start. The same for disarming.

I think most people would use an arming switch because:
- preference to use a switch for arming instead of a stick motion (they might be used to it because they used another autopilot)
- it avoids accidentially triggering it in-air with a certain stick motion
- there is no delay, it reacts immediately